src/client/components/atoms/burgerButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import StateContext from '../../../utils/stateContext';
+import ExpandMenu from '../expand';
+import './styles.scss';
+
+const BurgerButton = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { user } = useContext(StateContext);
+
+  const handleToggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+  return (
+    <div>
+      <button className={classNames(`burger ${isOpen ? 'open' : ''}`)} onClick={handleToggleMenu}>
```

Comment:
Bad use of BEM pattern and dynamic classes, you don't need a ternary operator nor template string

suggestion:
`<button className={classNames("burger", { "burger--open": isOpen })} onClick={handleToggleMenu}>`